### PR TITLE
PCHR-3937: Style changes for the Main Menu in smaller devices

### PR DESCRIPTION
**Pull request by deb1990**
## Overview
As part of this PR the following changes had been implemented
1. The styles for the selected item of the list has been changed.
2. Removed the styles to highlight HOME in grey.
3. Hiding the HR ADMIN button in mobile devices(Smaller than 768px in width).
4. In Tablets the HR ADMIN button is made smaller and center aligned.

## Before
**Mobile**
2018-07-09 at 3 52 pm

**Tablets**
2018-07-09 at 3 53 pm

## After
**Mobile**
2018-07-09 at 3 54 pm

**Tablets**
2018-07-09 at 6 32 pm

## Tests Done
Tested manually on

Chrome (macOS)
Mobile and Tablet Emulator in Chrome (macOS)

### CODE REVIEW

File: civihr_default_theme/scss/components/_header.scss [hunk 1]
```diff
@@ -215,16 +221,23 @@
     .menu-link-hr-admin {
       border: 1px solid $gray-darker;
       line-height: 50px;
-      margin: 25px;
+      margin: 25px auto;
       padding: 0;
       text-align: center;
+      width: 300px;
```
Comment (igorpavlov-zz) [inline civihr_default_theme/scss/components/_header.scss]: I don't think we should set a fixed width, can we make it `display: inline-block` instead?

File: civihr_default_theme/scss/components/_header.scss [hunk 2]
```diff
@@ -194,8 +189,19 @@
...
     flex-direction: column;
 
-    a {
+    > li {
       border-bottom: 1px solid $crm-grizzly;
+
+      &.active { /* stylelint-disable-line selector-no-qualifying-type */
+        background-color: $gray-light;
+
+        &::after { /* stylelint-disable-line max-nesting-depth */
```
Comment (igorpavlov-zz) [inline civihr_default_theme/scss/components/_header.scss]: Would `&.active::after` require `stylelint-disable-line max-nesting-depth`?

**Review by igorpavlov-zz — CHANGES_REQUESTED** [review]

File: civihr_default_theme/scss/components/_header.scss (hunk 1 above)
Comment (deb1990) [reply civihr_default_theme/scss/components/_header.scss]: Removing the width completely works too. Please check the updated screenshots.

File: civihr_default_theme/scss/components/_header.scss (hunk 2 above)
Comment (deb1990) [reply civihr_default_theme/scss/components/_header.scss]: Yes. Fixed.

**Review by igorpavlov-zz — APPROVED** [review]

**Pull request merged**